<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memory Analyzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainVerticalLayout">

    <item>
     <layout class="QHBoxLayout" name="topControlsLayout">
      <item>
       <layout class="QVBoxLayout" name="selectorLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font><family>Consolas</family><italic>true</italic></font>
          </property>
          <property name="text">
           <string>Select Input Method -</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="modeComboBox">
          <property name="font">
           <font><family>Consolas</family><bold>true</bold></font>
          </property>
          <item>
           <property name="text"><string>Observe Running Program</string></property>
          </item>
          <item>
           <property name="text"><string>Enter PID</string></property>
          </item>
         </widget>
        </item>
       </layout>
      </item>

      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <widget class="QWidget" name="page">
         <layout class="QHBoxLayout" name="layoutPage1">
          <item>
           <widget class="QLineEdit" name="processNameLineEdit">
            <property name="placeholderText">
             <string>Search process name...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QHBoxLayout" name="layoutPage2">
          <item>
           <widget class="QLineEdit" name="pidLineEdit">
            <property name="placeholderText">
             <string>Enter PID number</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>

      <item>
       <widget class="QPushButton" name="scanButton">
        <property name="minimumSize">
         <size><width>100</width><height>40</height></size>
        </property>
        <property name="font">
         <font><family>Consolas</family><pointsize>12</pointsize><italic>true</italic><bold>true</bold></font>
        </property>
        <property name="text"><string>Scan</string></property>
       </widget>
      </item>
     </layout>
    </item>

    <item>
     <widget class="QLabel" name="infoLabel">
      <property name="text"><string>Ready to analyze...</string></property>
      <property name="alignment"><set>Qt::AlignCenter</set></property>
     </widget>
    </item>

    <item>
     <widget class="QWidget" name="memoryBarPlaceholder" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      </widget>
    </item>

   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
